.github/workflows/nodejs.yml

```diff
@@ -20,6 +20,8 @@ jobs:
     - name: npm install, build, and test
       run: |
         npm ci
-        npm test
+        TS_NODE_TRANSPILE_ONLY=true time npm test
```

Comment:
this is the fix, found in https://github.com/TypeStrong/ts-node/issues/754.

ideally we should update tsconfig.json